### PR TITLE
update APY to show 0%

### PR DESCRIPTION
**Pull request by Jibz-Mysten**
## Description 

Describe the changes or additions included in this PR.

Show 0% APY and -- when APY is unavailable 
Fix StakeCard padding

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [x] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

**Review by pchrysochoidis — APPROVED** [review]

File: apps/wallet/src/ui/app/staking/stake/ValidatorFormDetail.tsx [hunk 1]
```diff
@@ -155,7 +159,7 @@ export function ValidatorFormDetail({
                                 weight="semibold"
                                 color="gray-90"
                             >
-                                {apy > 0 ? `${apy}%` : '--'}
+                                {apy === null ? '--' : `${apy}%`}
```
Comment (Jordan-Mysten) [inline apps/wallet/src/ui/app/staking/stake/ValidatorFormDetail.tsx:162]: Seems like this logic is really consistent, can we make a format function that we can share?

File: apps/wallet/src/ui/app/components/receipt-card/StakeTxnCard.tsx [hunk 2]
```diff
@@ -30,7 +30,11 @@ export function StakeTxnCard({ event }: StakeTxnCardProps) {
         system?.activeValidators?.length || null
     );
 
-    const apy = rollingAverageApys?.[validatorAddress] || 0;
+    const apy =
+        rollingAverageApys?.[validatorAddress] ||
+        rollingAverageApys?.[validatorAddress] === 0
```
Comment (Jordan-Mysten) [inline apps/wallet/src/ui/app/components/receipt-card/StakeTxnCard.tsx:35]: Just do `?? null`, which will handle the `0` case.

File: apps/wallet/src/ui/app/staking/stake/ValidatorFormDetail.tsx [hunk 3]
```diff
@@ -82,7 +82,11 @@ export function ValidatorFormDetail({
         );
     }, [stakeData, system, totalValidatorsStake, validatorAddress]);
 
-    const apy = rollingAverageApys?.[validatorAddress] || 0;
+    const apy =
+        rollingAverageApys?.[validatorAddress] ||
+        rollingAverageApys?.[validatorAddress] === 0
```
Comment (Jordan-Mysten) [inline apps/wallet/src/ui/app/staking/stake/ValidatorFormDetail.tsx:87]: Same here, remove the `0` case and just do `?? null`

File: apps/wallet/src/ui/app/components/receipt-card/StakeTxnCard.tsx (hunk 2 above)
Comment (Jibz-Mysten) [reply apps/wallet/src/ui/app/components/receipt-card/StakeTxnCard.tsx:35]: Just saw this, I will update the pr

**Pull request merged**